code/modules/background/origins/origins/vaurca/zora.dm

```diff
@@ -0,0 +1,53 @@
+/decl/origin_item/culture/zora
+	name = "Zo'ra Hive"
+	desc = "I FUCKING LOVE ROLEPLAYING AS A BUG: I LOVE PLAYING A SPECIES JUST TO NOT HAVE TO ROLEPLAY EMOTIONS"
```

Comment:
Seriously?

Reply:
Outdated, this was a joke placeholder description.